en/Obsidian/Community code of conduct.md

```diff
@@ -2,24 +2,40 @@
 
 These are the behaviors we do not tolerate.
 
-### Major offenses
+### Serious offenses
+
+Once verified, these offenses will result in an immediate ban from our community.
```

Comment:
I'd cut the "Once verified, " here and below. I think reasonable people would assume due diligence across the implementation of the entire CoC.